tests/rules/test.css.js

```diff
@@ -0,0 +1,31 @@
+import { BAD_URL_RX } from 'rules/css';
+import { singleLineString } from 'utils';
+
+
+describe('BAD_URL_RX', () => {
+
+  it('should match remote urls', () => {
+    assert.ok(BAD_URL_RX.test('url(http://foo/com)'));
+    assert.ok(BAD_URL_RX.test('url(https://foo/com)'));
```

Comment:
What about `//foo.com`? Not sure they work in add-ons but if they do we should test for them.

Also why the `/` in the string instead of a `.`?